js/SearchResult.js

```diff
@@ -0,0 +1,47 @@
+import {fetchData} from "./utils.js"
+const URLForNameAndSymbol = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=`
+const QuaryURLForNameAndSymbol = `&limit=10&exchange=NASDAQ`;
+
+
+export class SearchResults {
+    constructor (companies){
+        this.companies = companies;
+    }
+
+    async renderResults() {
+        this.companies.innerHTML = "";
+        const companyResultResponse = await fetchData(URLForNameAndSymbol + document.getElementById('companySearch').value + QuaryURLForNameAndSymbol);
+        const unOrderedList = document.createElement("ul");
+        const numOfResluts = Math.min(companyResultResponse.length, 10);
+
+        for (let i = 0; i < numOfResluts; i++) {
+          let symbol = companyResultResponse[i].symbol;
+          const responseJsonFromCompanyData = await fetchData(`https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/${symbol}`);
+        
+          const companyLogo = document.createElement("img");
+          companyLogo.className = "small_img";
+          companyLogo.setAttribute("src", responseJsonFromCompanyData.profile.image);
+        
+          const listItem = document.createElement("li");
+          let searched = document.getElementById("companySearch").value.trim();
+          let text = companyResultResponse[i].name;
+          let re = new RegExp("(" + searched + ")","ig");
+          let newText = text.replace(re, `<mark>$1</mark>`);
+          text = newText;
+          const Item = document.createTextNode(text + " (" + companyResultResponse[i].symbol + ") ("+ Number(responseJsonFromCompanyData.profile.changesPercentage).toFixed(2)+")");
```

Comment:
This line is wayyyy to long. also, notice that you declared a const with Capital letter.

Store things like companyResultResponse[i].symbol, responseJsonFromCompanyData.profile.changesPercentage 
In variables and call them here.